src/main/webapp/i18n/de/aboutUs.json

```diff
@@ -16,20 +16,20 @@
             "title": "Artemis unterstützt folgende Features für Klausuren",
             "detail": {
                 "supportedExerciseTypes": "Alle Aufgabentypen sind auch in der Klausur verfügbar",
-                "randomSelectionOfVariants": "Mehrere Varianten von Aufgaben können erstellt und den Studenten zufällig zugewiesen werden",
+                "randomSelectionOfVariants": "Mehrere Varianten von Aufgaben können erstellt und den Studierenden zufällig zugewiesen werden",
                 "fullFeatureOverview": {
                     "description": "Gesamter Überblick über alle Features:",
-                    "studentLink": "Überblick für Studenten",
-                    "instructorLink": "Überblick für Dozenten"
+                    "studentLink": "Überblick für Studierende",
+                    "instructorLink": "Überblick für Lehrpersonen"
```

Comment:
I really like the changes overall, but I am not sure about the use of 'Lehrpersonen'. The problem is that this word does not exist. I googled quite a bit, its apparently only used a historical swiss-word. I think we should be strict with only using gender inclusive language, but not invent new words or use words that are just plain wrong in this context. 😃  If we want to be super strict, we have to use 'Dozenten und Dozentinnen' or 'Dozentenschaft' or 'Lehrende' or maybe someone else has an even better idea?  😄

Reply:
> I really like the changes overall, but I am not sure about the use of 'Lehrpersonen'. 

Thanks for the review, I got your point... I found it here https://geschicktgendern.de/#L. But I agree, that `Lehrende` is a better term. I will change it.

Reply:
I looked in up the multiple official German language sites, like duden.de Seems like some sites invent / misuse some words, I ran into the same issue before. Usually better to double check the words.  As a recommendation: duden.de has lots of gender related advice if you go to the female version of a word. I attached an example link: [https://www.duden.de/rechtschreibung/Dozentin](https://www.duden.de/rechtschreibung/Dozentin)

Reply:
ah perfect, thanks! I think will do the `Lehrende`-Refactoring today!

Reply:
done via fc4d75f